git-to-obs-package

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
Fair point, as we want to use it for https://github.com/openSUSE/libstorage in this same PBI. So I have split it off into https://github.com/openSUSE/obs-package-from-git